README.md

```diff
@@ -53,9 +70,214 @@ This is the simplest case. Get the script in whatever way you prefer and include
+
+* `'in-page'` or `brightcovePlayerLoader.EMBED_TYPE_IN_PAGE`: Also referred to as the advanced embed code, this injects the player directly into the top-level web page.
+* `'iframe'` or `brightcovePlayerLoader.EMBED_TYPE_IFRAME`: Also referred to as the basic embed code, this injects the player as an `<iframe>` element.
+
+#### `onEmbedCreated`
+* *Type:* Function
+* *Default:* `undefined`
+
+A callback used to customize the embed element (either `video-js` element or an `iframe` element) *before* it is inserted into the DOM or customized as a result of `embedOptions` and before the player is downloaded and initialized.
+
+The return value of this function is ignored; the embed element should be mutated.
+
+Potential use-cases are adding/removing attributes or adding child elements, such as sources or tracks.
+
+#### `options`
+* *Type:* Object
+* *Default:* `undefined`
+
+A [Video.js options object][vjs-options] to pass during the player creation process. These options will take precedence over any settings specified in the Brightcove Player configuration.
+
+> **NOTE:** _Cannot be used with iframe embeds!_
+
+#### `playerId`
+* *Type:* string
+* *Default:* `'default'`
+
+A Brightcove Player ID.
+
+#### `playlistId`
+* *Type:* string | number
+* *Default:* `undefined`
+
+A Video Cloud video ID or [reference ID][bc-ref-id].
+
+#### `playlistVideoId`
+* *Type:* string | number
+* *Default:* `undefined`
+
+A Video Cloud video ID that would be found in the resulting playlist specified by `playlistId`. This parameter is ignored if `playlistId` is missing.
+
+#### `Promise`
+* *Type:* Function
+* *Default:* `window.Promise`
+
+Used to explicitly provide a `Promise` implementation. If provided, this will be used in lieu of any global `Promise`.
+
+#### `refNode`\*
+* **REQUIRED**
+* *Type:* Element | string
+* *Default:* `undefined`
+
+The DOM element into which the player will be embedded. If not provided as a DOM element, it can be provided as a string, which will be passed to `document.querySelector`.
+
+#### `refNodeInsert`
+* *Type:* string
+* *Default:* `'append'`
+
+The manner in which the player will be inserted relative to the reference DOM element (specified by `refNode`). This parameter's value must be one of the following:
+
+* `'append'` or `brightcovePlayerLoader.REF_NODE_INSERT_APPEND`: The player will be the last child of the reference node.
+* `'prepend'` or `brightcovePlayerLoader.REF_NODE_INSERT_PREPEND`: The player will be the first child of the reference node.
+* `'before'` or `brightcovePlayerLoader.REF_NODE_INSERT_BEFORE`: The player will be previous sibling of the reference node.
+* `'after'` or `brightcovePlayerLoader.REF_NODE_INSERT_AFTER`: The player will be the next sibling after the reference node.
+* `'replace'` or `brightcovePlayerLoader.REF_NODE_INSERT_REPLACE`: The reference node will be removed and replaced by the player.
+
+#### `videoId`
```

Comment:
what about `search`/`q`

Reply:
Good point. I forgot those are available on the embed code now.

Reply:
Added `catalogSearch` and `catalogSequence` options.